lib/resources/r4/related_person_patch.yaml

```diff
@@ -2,6 +2,33 @@
 name: Related Person
 field_name_base_url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson
 operations:
+  - name: test-identifier-id
+    path: /identifier/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the value of the `Identifier.id` of the identifier at the given <code>{index}</code> in the list of related person identifiers.
+    example: |
+      {
+        "path": "/identifier/0/id",
+        "op": "test",
+        "value": "CI-490050200-0"
```

Comment:
Is this a C1941 id?

Reply:
updated id from C1941 in this commit https://github.com/cerner/fhir.cerner.com/pull/548/commits/282631d312acb0eeb67f1c520a3471e8724173ff